mne/time_frequency/tfr.py

```diff
@@ -1127,11 +1127,9 @@ def plot(self, picks, baseline=None, mode='mean', tmin=None, tmax=None,
 
     def _onselect(self, eclick, erelease, baseline, mode, layout):
         """Handle rubber band selector in channel tfr."""
-        import matplotlib.pyplot as plt
         from ..viz import plot_tfr_topomap
         if abs(eclick.x - erelease.x) < .1 or abs(eclick.y - erelease.y) < .1:
             return
-        plt.ion()  # turn interactive mode on
```

Comment:
I don't think we should mess with matplotlib interactive status. Messing with global states is not a nice thing for a package to do.

It would be better to document any functions that need interactivity, telling people to activate interactive mode (maybe with docs somewhere about how to do it).